src/applications/facility-locator/tests/e2e/mobile.cypress.spec.js

```diff
@@ -71,7 +71,7 @@ describe('Mobile', () => {
     });
   });
 
-  it('should render in mobile layouts, clear input checks and tabs actions work', () => {
+  it.skip('should render in mobile layouts, clear input checks and tabs actions work', () => {
```

Comment:
This test should not have been skipped.

Reply:
the logs from this commit would suggest otherwise.
logs: https://app.circleci.com/pipelines/github/department-of-veterans-affairs/vets-website[…]2/workflows/4ecb4179-5f78-4138-adf3-54c08592c906/jobs/212854
commit: https://github.com/department-of-veterans-affairs/vets-website/runs/2079772844

Reply:
i will create a PR to revert.